.changeset/short-walls-read.md

```diff
@@ -0,0 +1,5 @@
+---
+"@launchpad-ui/icons": minor
```

Comment:
```suggestion
"@launchpad-ui/icons": patch
```
We can set to patch since [we follow 0.x versioning](https://github.com/launchdarkly/launchpad-ui/blob/main/CONTRIBUTING.md#versioning)